extensions/amp-story/1.0/test/test-amp-story.js

```diff
@@ -1837,5 +1840,39 @@ describes.realWin(
         ).to.be.false;
       });
     });
+
+    describe('experiment for amp-story-load-first-page-only', () => {
+      let pages;
+      beforeEach(async () => {
+        toggleExperiment(win, 'amp-story-load-first-page-only', true);
+        pages = await createStoryWithPages(2, ['page-1', 'page-2'], false);
+        env.sandbox.stub(story, 'mutateElement').callsFake((fn) => fn());
+      });
+
+      it('should only load the active page with distance=0', async () => {
+        story.buildCallback();
+        await story.layoutCallback();
+
+        // Check page 0 is loaded with distance 0.
+        expect(pages[0].getAttribute('distance')).to.be.equal('0');
+      });
+
+      it('should load the inactive pages after the active page is loaded with distance=1', async () => {
```

Comment:
Suggestion so it carries more context / is more accurate, feel free to use your own words

```suggestion
      it('should position the inactive page so it preloads after the active page is loaded', async () => {
```